### PR TITLE
ci: add github workflow

### PR DESCRIPTION
**Pull request by x1ddos**
This is an alternative to Travis CI.
It runs the same scripts in docker on Ubuntu 18.04.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Clone the repo
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: vars
+        run: echo "---${{github.head_ref}}---"
```
Comment (x1ddos) [inline .github/workflows/ci.yml:15]: One thing I'm not sure of is whether `github.head_ref` is the same as `TRAVIS_BRANCH` when a PR is sent from another repo, but I guess we'll find out soon.

**Comment by x1ddos** [comment]
Github was first:

image

File: .ci/run-container [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```
Comment (benma) [inline .ci/run-container:1]: `run-container` -> would be nice to keep `ci` in the name, `run_container` sounds a bit too generic

Comment (x1ddos) [reply .ci/run-container]: ok. renamed to `run-container-ci`.

**Comment by x1ddos** [comment]
:+1: to run them both for a while

**Pull request merged**